### PR TITLE
Group by usecase

### PR DESCRIPTION
**Pull request by samdark**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ✔️
| Fixed issues  | -

### CODE REVIEW

File: config/params.php [hunk 1]
```diff
@@ -102,8 +102,7 @@
             ],
         ],
         'annotated-entity-paths' => [
-            '@src/Entity',
-            '@src/Blog/Entity',
+            '@src',
```
Comment (BoShurik) [inline config/params.php:105]: Is there a performance impact? Cycle needs to check all classes in that case

Comment (samdark) [reply config/params.php:105]: Sure there is but that's done only once. Then schema is cached.

Comment (BoShurik) [reply config/params.php:105]: Is it possible to warmup cache?

File: src/User/Console/CreateCommand.php [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 declare(strict_types=1);
 
-namespace App\Command\User;
+namespace App\User\Console;
```
Comment (BoShurik) [inline src/User/Console/CreateCommand.php:5]: Why commands inside a `Console` namespace, but controllers does not have own `Controller`?
Maybe it worse to add additional level of context: `Console`, `Controller`, `Entity`, `Repository` etc

Comment (samdark) [reply src/User/Console/CreateCommand.php:5]: Yes, makes sense when there are multiple objects of the same type.

Comment (BoShurik) [reply src/User/Console/CreateCommand.php:5]: Same for https://github.com/yiisoft/yii-demo/tree/group-by-usecase/src/Blog

File: config/params.php (hunk 1 above)
Comment (samdark) [reply config/params.php:105]: Yes, with `./vendor/bin/yii cycle/schema/rebuild`.

**Comment by rustamwin** [comment]
[Group classes by use-case · Issue #3 · yiisoft/app-api](https://trello.com/c/cWSKLufg/156-group-classes-by-use-case-issue-3-yiisoft-app-api)

**Pull request merged**